documentation/jetty-documentation/src/main/asciidoc/operations-guide/modules/module-threadpool.adoc

```diff
@@ -16,15 +16,15 @@
 
 The `threadpool` module allows you to configure the server-wide thread pool.
 
-The thread pool creates threads on demand up to `maxThreads`, and idle them out if they are not used.
+The thread pool creates threads on demand up to `maxThreads`, and idles them out if they are not used.
 
 Since Jetty uses the thread pool internally to execute critical tasks, it is not recommended to constrain the thread pool to small values of `maxThreads` with the purpose of limiting HTTP request concurrency, as this could very likely cause a server lockup when Jetty needs to run a critical task but there are no threads available.
 Start with the default value of `maxThreads`, and tune for larger values if needed.
 
-The module file is `$JETTY_HOME/modules/threadpool.mod`:
+The module properties to configure the thread pool are:
```

Comment:
I think you can just drop this line and/or merge it with line 30

Reply:
It's a pattern that I used for all the modules docs, so I'd like to keep it.

Reply:
Then line 30 is redundant.